hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -232,7 +243,16 @@ public int read(String table, String key, Set<String> fields, HashMap<String,Byt
                     g.addColumn(_columnFamilyBytes, Bytes.toBytes(field));
                 }
             }
-            r = _table.get(g);
+            if (multiGetBatchSize > 0) {
+              multiGets.add(g);
+              if (multiGetBatchSize == multiGets.size()) {
+                r = _table.get(multiGets);
```

Comment:
I can fix other comments. But the latency distribution part am not sure how to handle with multiGets.  So till the multiGet batch size is reached the latency distribution is going to be high since we don do the actual operation but once the batch size is reached then the multiGets latency gets included.